### PR TITLE
Fix AutoYaST schema for SLE 12 GA

### PR DESCRIPTION
**Pull request by imobach**

### CODE REVIEW

File: package/yast2-kdump.changes [hunk 1]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon Nov  9 13:06:10 UTC 2015 - igonzalezsosa@suse.com
+
+- Fix AutoYaST profile schema (bsc#805275)
+- 3.1.20
```
Comment (lslezak) [inline package/yast2-kdump.changes]: It seems that `3.1.20` has been already used in `master`, it should be `3.1.19.1` I guess...

**Comment by lslezak** [comment]
Otherwise LGTM...

**Comment by imobach** [comment]
@lslezak You're right! Thanks!

**Comment by lslezak** [comment]
LGTM

**Comment by lslezak** [comment]
Still LGTM :wink:

**Comment by imobach** [comment]
Thanks again :)

**Pull request merged**